### PR TITLE
scroll to bottom on channel change

### PR DESCRIPTION
**Pull request by khubo**

### CODE REVIEW

File: app/containers/messages.js [hunk 1]
```diff
@@ -50,123 +56,129 @@ function MessagesContainer (props) {
...
-                </div>
-              )}
-              {item}
-            </div>
-          )
-        })}
-      </div>
+          })}
+        </div>
+        <div
```
Comment (khubo) [inline app/containers/messages.js:177]: added this div so it can be ref to scrolled down to

**Comment by cblgh** [comment]
yassssss 🙌 🙌

**Comment by nikolaiwarner** [comment]
NICE! yay for hooks! i've tested it out and it's working great!

File: app/containers/messages.js (hunk 1 above)
Comment (cblgh) [reply app/containers/messages.js:177]: @khubo maybe this comment can be inlined with the source code, as well? i bet someone in the future will be like "what is this div here for?!"

File: app/containers/messages.js [hunk 2]
```diff
@@ -50,123 +56,130 @@ function MessagesContainer (props) {
...
-                </div>
-              )}
-              {item}
-            </div>
-          )
-        })}
-      </div>
+          })}
+        </div>
+        {/* placeholder div for scrolling down to the bottom */}
```
Comment (khubo) [inline app/containers/messages.js:177]: @cblgh done ^_^

**Comment by cblgh** [comment]
@khubo you alright with this being merged? :3

**Comment by khubo** [comment]
@cblgh yupp

**Comment by cblgh** [comment]
heyoooooo, thanks again! 🖤

**Pull request merged**